grisera_api/participant_state/participant_state_service.py

```diff
@@ -13,6 +18,8 @@ class ParticipantStateService:
     """
```

Comment:
Missing attributes in the comment:
appearance_service (AppearanceService): Service to manage appearance models
personality_service (PersonalityService): Service to manage personality models